tests/integration/test_quorum_inserts/test.py

```diff
@@ -46,7 +46,7 @@ def started_cluster():
 
 
 def test_simple_add_replica(started_cluster):
-    zero.query("DROP TABLE IF EXISTS test_simple ON CLUSTER cluster")
+    zero.query("DROP TABLE IF EXISTS test_simple ON CLUSTER cluster SYNC")
```

Comment:
I think we can ignore this test and remove the changes. The linked pr tries to address the issues.

Reply:
Agreed